src/main/java/org/jabref/gui/groups/GroupColorPicker.java

```diff
@@ -0,0 +1,58 @@
+package org.jabref.gui.groups;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+import javafx.scene.paint.Color;
+
+import org.jabref.gui.icon.IconTheme;
+
+public class GroupColorPicker {
+    public static boolean useRandom = false;
+    private static final Random RANDOM = new Random();
+    private static final List<Color> TOP_LEVEL_COLORS = new ArrayList<>();
+    private static final int TRIES = 10000;
+
+    // Get most distinct color
```

Comment:
Please remove that comment - it is the same as the method name and thus obsolete

(and this is a good thing, because this is an indication that the method names are well-chosen)